app/api/dao/admin.py

```diff
@@ -38,6 +39,7 @@ def assign_new_user(assigner_user_id, data):
         return {"message": "User does not exist."}, 404
 
     @staticmethod
+    @email_verification_required
     def revoke_admin_user(revoker_user_id, data):
```

Comment:
```suggestion
    def revoke_admin_user(user_id, data):
```

Reply:
don't forget to fix the doc strings to reflect the new argument name

Reply:
Done